pages/task/read/views/pil.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment } from 'react';
+import { Snippet } from '@asl/components';
+import { dateFormat, procedureDefinitions } from '../../../../constants';
+import { connect } from 'react-redux';
+import format from 'date-fns/format';
+import addYears from 'date-fns/add_years';
+
+const Pil = ({ profile }) => {
+  const pil = profile.pil;
+  const formatDate = date => format(date, dateFormat.ddMMyyyy);
+  return (
+    <Fragment>
+      <h2><Snippet>pil.training.title</Snippet></h2>
+      { profile.certificates.length && profile.certificates.map((certificate, index) => (
+        <div key={index}>
+          <Fragment>
+            <h3><Snippet>pil.training.certificate.details</Snippet></h3>
+            <Snippet>pil.training.certificate.number</Snippet><span>:</span> {certificate.certificateNumber}<br/>
+            <Snippet>pil.training.certificate.awarded</Snippet><span>:</span> {formatDate(certificate.passDate)}<br/>
+            <Snippet>pil.training.certificate.expiry</Snippet><span>:</span> {formatDate(addYears(certificate.passDate, 5))}<br/>
+            <Snippet>pil.training.certificate.body</Snippet><span>:</span> {certificate.accreditingBody}<br/>
+            <Snippet>pil.training.certificate.file</Snippet><span>:</span> <br/>
```

Comment:
Can we use a `<dl>` instead of random line breaks please?

At the absolute worst use `<p>` but a `<dl>` is best.